cmd/example.toml

```diff
@@ -33,4 +33,7 @@ reserved = [
 
 [checks.namespace]
 [[checks.namespace.patterns]]
-py = "^idl\\."
\ No newline at end of file
+py = "^idl\\."
+
+[checks.types]
+disallowed = []
```

Comment:
We used "restricted" in #98 and "disallowed" here. I think we should use be consistent.

"disallowed" feels a little clearer to me, but I don't feel strongly, so maybe you and @swatkatz can decide.